android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,14 @@
+<resources>
+  <string name="app_name">Movement-Vites</string>
+  <string name="dev_activity_name">Exponent Dev</string>
+  <string name="preference_file_key">host.exp.exponent.SharedPreferences</string>
+  <string name="error_header">Something went wrong.</string>
+  <string name="error_default_client">Sorry about that. You can go back to Expo home or try to reload the Experience.</string>
```

Comment:
Im guessing these error messages need to be re-written for our application? what is all this experience stuff?

Reply:
realizing this might be part of expo